### PR TITLE
[COURSE] Add UW–Madison CS571 for Web/Mobile Development

### PR DESCRIPTION
**Pull request by alicia-lyu**
This course provides a comprehensive but concise introduction to the best practices of React front-end development and React Native mobile development. It focuses on the latest versions of React and React Native and is updated every semester. It is a valuable resource for tackling the complexities of front-end development.

The course also offers a good training ground. Be prepared for a significant workload throughout the semester. If you are unsure about how to write React code when doing homework, I recommend spending extra time carefully reading the relevant chapters on react.dev before diving in.

### CODE REVIEW

File: docs/Web开发/CS571.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## 课程简介
+
+- 所属大学：威斯康星大学麦迪逊分校（University of Wisconsin, Madison）
+- 先修要求：CS400（高级Java，但个人觉得先修不必要）
+- 编程语言：JavaScript/HTML/CSS
+- 课程难度：🌟🌟🌟
+- 预计学时：每周2小时（讲座）+ 每周4-10小时（作业），持续12周
+
+该课程提供了React前端开发和React Native移动端开发的最佳实践介绍，完整又简洁。重点关注React和React Native的最新版本，课程网站每学期都会更新。对于各门工具迭出的前端开发难能可贵。
```
Comment (PKUFlyingPig) [inline docs/Web开发/CS571.md:11]: Please add a space between the English and Chinese words. For example, "该课程提供了 React 前端开发". Please refer to [this tutorial](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-Hans.md).

**Comment by PKUFlyingPig** [comment]
Thanks for your contribution, this is a great course! Please remember to add an entry in the “Web开发” section in [mkdocs.yml](https://github.com/PKUFlyingPig/cs-self-learning/blob/master/mkdocs.yml) to make your course available on the csdiy.wiki website.

File: docs/Web开发/CS571.md (hunk 1 above)
Comment (alicia-lyu) [reply docs/Web开发/CS571.md]: Please see my latest commit.

**Comment by alicia-lyu** [comment]
> Thanks for your contribution, this is a great course! Please remember to add an entry in the “Web开发” section in [mkdocs.yml](https://github.com/PKUFlyingPig/cs-self-learning/blob/master/mkdocs.yml) to make your course available on the csdiy.wiki website.

Done! Thank you for sharing and building the whole infrastructure!

**Pull request merged**